packages/frameworks/frameworks.json

```
@@ -1,4 +1,32 @@
 [
+  {
+    "name": "Blitz.js",
+    "slug": "blitzjs",
+    "demo": "https://blitzjs.now-examples.now.sh",
+    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/blitz.svg",
+    "tagline": "Blitz.js: The Fullstack React Framework",
+    "description": "A brand new Blitz.js app: the output of running `blitz new`",
+    "website": "https://blitzjs.com",
+    "detectors": {
+      "every": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"blitz\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`npm run build` or `blitz build`"
+      },
+      "devCommand": {
+        "value": "blitz start"
```

How do we pass the PORT here? Is there a flag?

---

Same as Next, `-p`

---

```suggestion
        "value": "blitz start --port $PORT"
```

Great, then lets use the port flag like Next.js does below.